util/getSkills.js

```diff
@@ -0,0 +1,34 @@
+// order of keys in this array determines the order of the return array 
+const skillKeys = [
```

Comment:
Really like the use of SRP (single responsibility principle) here. It is easy to read and these keys can easily be updated in the future. 👍